Python/Tests/Core.UI/TestExplorerTests.cs

```diff
@@ -122,6 +169,31 @@ public class TestExplorerTests {
             return workspaceFolderPath;
         }
 
+        private static void DebugTest(PythonVisualStudioApp app, TestInfo test, TestInfo[] tests) {
```

Comment:
i'm only debugging a single test, but I still need all tests so that I can wait for discovery to finish and check it discovered all the tests.